bootstrap/bungeecord/src/main/java/org/geysermc/platform/bungeecord/GeyserBungeeConfiguration.java

```diff
@@ -62,6 +65,11 @@ public GeyserBungeeConfiguration(File dataFolder, Configuration config) {
         }
     }
 
+    public void loadFloodgate(GeyserBungeePlugin plugin) {
+        Plugin floodgate = plugin.getProxy().getPluginManager().getPlugin("floodgate-bungee");
```

Comment:
Pass the plugin in the constructor instead of retrieving it from the plugin manager directly.